docs/contributing/dev_environment.md

```diff
@@ -0,0 +1,176 @@
+    ```
+
+    !!! note "You can also clone using ssh which is more secure but requires more setup. Because of the additional setup, cloning using https as shown above is recommended"
+
+You should now have a new folder in your `hackforla` folder called `peopledepot`. Verify this by changing into the new directory:
+
+```bash
+cd peopledepot
+```
+
+### Verify and set up remote references
+
+Verify that your local cloned repository is pointing to the correct `origin` URL (that is, the forked repo on your own GitHub account):
+
+```bash
+git remote -v
+```
+
+You should see `fetch` and `push` URLs with links to your forked repository under your account (i.e. `https://github.com/<your_GitHub_user_name>/peopledepot.git`). You are all set to make working changes to the project on your local machine.
+
+However, we still need a way to keep our local repo up to date with the deployed project. To do so, you must add an upstream remote to incorporate changes made while you are working on your local repo. Run the following to add an upstream remote URL & update your local repo with recent changes to the `hackforla` version:
+
+```bash
+git remote add upstream https://github.com/hackforla/peopledepot.git
+git fetch upstream
+```
+
+After adding the upstream remote, you should now see it if you again run `git remote -v` :
+
+```bash
+origin  https://github.com/<your_GitHub_user_name>/peopledepot.git (fetch)
+origin  https://github.com/<your_GitHub_user_name>/peopledepot.git (push)
+upstream        https://github.com/hackforla/peopledepot.git (fetch)
+upstream        https://github.com/hackforla/peopledepot.git (push)
+```
+
+## Build and run using Docker locally
+
+1. Make sure the Docker service is running
+
+    === "Docker (Engine)"
+        ```bash
+        sudo systemctl status docker
+        ```
+
+        It will show `Active: active (running)` if it's running.
+
+    === "Docker Desktop"
+        1. Start Docker Desktop
+        1. Run `docker container ls` to verify Docker Desktop is running. If it is not running you will get the message: `Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?`
+
+1. Create an .env.docker file from .env.docker-example
+
+    ```bash
+    cp ./app/.env.docker-example ./app/.env.docker
+    ```
+
+1. Build and run the project via the script (this includes running `docker-compose up`)
+
+    ```bash
+    ./scripts/buildrun.sh
+    ```
+
+1. Create a super user for logging into the web admin interface
+
+    ```bash
+    docker-compose exec web python manage.py createsuperuser --no-input
+    ```
+
+1. Browse to the web admin interface at `http://localhost:8000/admin/` and confirm the admin site is running. Use DJANGO_SUPERUSER_USERNAME and DJANGO_SUPERUSER_PASSWORD from .env.docker for credentials.
+
+See our documentation for [Working with Docker](http://localhost:8005/contributing/tools/docker/#working-with-docker) for more useful Docker commands.
```

Comment:
Line 165: is the link to Working with Docker page to be inserted later?

Reply:
Great catch! localhost is definitely a placeholder link that didn't get cleaned up.

Reply:
Fixed